runtime/doc/usr_41.txt

```diff
@@ -832,9 +832,8 @@ List manipulation:					*list-functions*
 	range()			return a List with a sequence of numbers
 	string()		String representation of a List
 	call()			call a function with List as arguments
-	index()			index of a value in a List or Blob
-	indexof()		index in a List or Blob where an expression
-				evaluates to true
+	index()			index of a value in a List
+	indexof()		index in a List where an expression evaluates to true
```

Comment:
This exceeds 78 columns. If you exceed this limit, you will need to use `gqq` or similar to break lines at the appropriate positions.